packages/react-reconciler/src/ReactFiberRoot.js

```diff
@@ -107,6 +118,13 @@ export function createFiberRoot(
     firstBatch: null,
     nextScheduledRoot: null,
   };
+
+  if (enableInteractionTracking) {
+    ((root: any): FiberRoot).interactionThreadID = getThreadID();
+    ((root: any): FiberRoot).memoizedInteractions = new Set();
+    ((root: any): FiberRoot).pendingInteractionMap = new Map();
```

Comment:
Can adding these later affect hidden class? I wonder if that or the number of fields itself could mess with some optimizations.

Reply:
Hmm! I don't know. If so, I could move the definition of `root` within the if/else to ensure that we always added the same number of fields based on the feature flag.